Realm/Realm/Exceptions/RealmInUseException.cs

```diff
@@ -0,0 +1,31 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+namespace Realms.Exceptions
+{
+    /// <summary>
+    /// An exception thrown when an an operation is being performed on an open Realm
+    /// that can only performed on closed Realm (i.e. Realm files).
```

Comment:
Not sure I understand the `i.e. Realm files` clarification there - did you mean something like "delete realm files"?

Reply:
I think so. I had something in mind but the result was not clear enough. Corrected with https://github.com/realm/realm-dotnet/pull/2401/commits/321d250e5c6935cf2c6388bc0c7314d4842816ee.